### DISCUSSION TITLE
Preserving structure of full trace without simplifications

### DISCUSSION THREAD

**Discussion by bubbleofnothing** [FeynCalc/feyncalc#286] [Q&A] [answered]
Hello - 
In older versions of Feyncalc, one could compute a Dirac trace without enforcing certain simplifications (i.e. Schouten identities, etc.). So, for example, in the old reference guide one could compute DiracTrace[GA[5, \[Mu], \[Nu], \[Alpha], \[Beta], \[Gamma], \[Delta]] and obtain all 15 terms from the trace, which is what would get when replacing gamma5 with its definition in 4-dimensions. 

Currently, DiracTrace will always simplify giving only six terms. How to compute the full trace without simplifications? I've tried all the usual options. Additionally, is there a way to control what simplifications are made? For instance, if I wanted to ensure the result of the trace was fully symmetric in Lorentz indices?

**Comment by vsht** [comment, marked as answer]
Hi,

the 4D trace is calculated using an optimized algorithm, so there's no way to make it generate more terms.

If you need just the trace expression, then using the BMHV scheme and then converting to 4D should probably
give you what you want.

```
FCSetDiracGammaScheme["BMHV"]
DiracTrace[GAD[i1, i2, i3, i4, i5, i6, 5], 
  DiracTraceEvaluate -> True] // ExpandAll
```

**Reply by bubbleofnothing** [reply]
As a workaround, this does appear to give what I want. Thanks a lot.
